common/README.md

```diff
@@ -3,3 +3,6 @@
 ## common/sample_flows
 Some Metaflow flows that could be used to test drive your Metaflow-on-Cloud stack. Some of these flows
 are used to drive end-to-end CI coverage internally at Outerbounds.
+
+## common/dags-sync
```

Comment:
What is ultimate goal of the things within this folder?  If it's general sync, should we call it `common/cloud-sync-tools` or something like that?  Or if it really for dags only (right now), let's call it `common/airflow-dags-sync`.

I would lean towards the latter - it's always easier to expand scope later in a more decisive manner.

Reply:
I like cloud-sync tools better. I will use that. I am already going to use this for the AWS template too.

Reply:
Plus that module will ideally even accommodate gcp too